app/templates/index.html

```diff
@@ -2,91 +2,90 @@
-            <div class="btn-group">
-              <a href="/{{ service.slug }}.html" class="btn btn-sm btn-outline-primary">
-                Details
-              </a>
+      <div class="col-md-4 col-sm-12 col-12">
+        <div class="card mb-4 shadow bg-white rounded">
+          <img src="/static/images/{{ service.images[0] }}" class="card-img-top" alt="{{ service.title }}" height="225" width="100%">
+
+          <div class="card-body">
+            <h1><a href="/{{ service.slug }}.html">{{service.name}}</a></h1>
+            <p class="card-text">
+              {{ service.summary|striptags }}
+            </p>
+            <div class="d-flex justify-content-between align-items-center">
+              <div class="btn-group">
+                <a href="{{ url_for('home.services_detail', id=service.id, slug=service.slug) }}" class="btn btn-sm btn-outline-primary">
+                  Details
+                </a>
+              </div>
+              <small class="text-muted">tbd</small>
             </div>
-            <small class="text-muted">tbd</small>
           </div>
-        </div>
 
+        </div>
       </div>
-    </div>
     {% else %}
-    No posts yet!
+      No posts yet!
     {% endfor %}
   </div>
 
 
 
-<div class="row justify-content-md-center">
+  <div class="row justify-content-md-center">
     <h1 class="header-title">Our Inventory</h1>
-</div>
+  </div>
 
-<div class="row">
+  <div class="row">
     {% for vehicle in content['vehicles'][:6] %}
-    <div class="col-md-4 col-sm-12 col-12">
-      <div class="card mb-4 shadow bg-white rounded">
-        <img src="{{ vehicle.cover }}" class="card-img-top" alt="{{ vehicle.slug }}" height="225" width="100%">
-
-        <div class="card-body">
-          <h1><a href="/{{ vehicle.slug }}.html">{{vehicle.slug}}</a></h1>
-          <p class="card-text">
-            {{ vehicle.summary|striptags }}
-          </p>
-          <div class="d-flex justify-content-between align-items-center">
-            <div class="btn-group">
-              <a href="/inventory/{{ vehicle.id }}" class="btn btn-sm btn-outline-primary">
-                Details
-              </a>
+      <div class="col-md-4 col-sm-12 col-12">
+        <div class="card mb-4 shadow bg-white rounded">
+          <img src="{{ vehicle.cover }}" class="card-img-top" alt="{{ vehicle.slug }}" height="225" width="100%">
+
+          <div class="card-body">
+            <h1><a href="/{{ vehicle.slug }}.html">{{vehicle.slug}}</a></h1>
+            <p class="card-text">
+              {{ vehicle.summary|striptags }}
+            </p>
+            <div class="d-flex justify-content-between align-items-center">
+              <div class="btn-group">
+                <a href="/inventory/{{ vehicle.id }}" class="btn btn-sm btn-outline-primary">
+                  Details
+                </a>
+              </div>
+              <small class="text-muted">tbd</small>
```

Comment:
I think there is no changes in this section right? just formatting?

Reply:
Yes